src/OrchardCore.Modules/OrchardCore.Media/Startup.cs

```diff
@@ -133,6 +133,7 @@ public override void ConfigureServices(IServiceCollection services)
             services.AddTransient<IConfigureOptions<ImageSharpMiddlewareOptions>, MediaImageSharpConfiguration>();
 
             services.AddImageSharp()
+                .SetMemoryAllocator<ArrayPoolMemoryAllocator>()
```

Comment:
This shouldn't be included. It's a mistake that I forgot to delete. The allocator is unused in the codebase.